v2.4/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -69,30 +69,11 @@ $ calicoctl get profile -o wide
 NAME                          TAGS
 k8s_ns.advanced-policy-demo   k8s_ns.advanced-policy-demo
 k8s_ns.default                k8s_ns.default
+k8s_ns.kube-public            k8s_ns.kube-public 
 k8s_ns.kube-system            k8s_ns.kube-system
 ```
 
-Because we've enabled isolation on the Namespace, the profile denies all ingress
-traffic and allows all egress traffic.
-
-```
-$ calicoctl get profile k8s_ns.advanced-policy-demo -o yaml
-- apiVersion: v1
-  kind: profile
-  metadata:
-    name: k8s_ns.advanced-policy-demo
-    tags:
-    - k8s_ns.advanced-policy-demo
-  spec:
-    egress:
-    - action: allow
-      destination: {}
-      source: {}
-    ingress:
-    - action: deny
-      destination: {}
-      source: {}
-```
+Because we've enabled isolation on the Namespace, all pods in this Namespace will be selected by Calicos "default-deny-if-selected" behavior.
```

Comment:
Might be better to say something like:

"Because all pods in the namespace are now selected, any traffic which is not explicitly allowed by a policy will be denied"